orttraining/orttraining/training_ops/cpu/aten_ops/aten_op.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/dlpack/dlpack_converter.h"
+#include "core/framework/op_kernel_context_internal.h"
+#include "orttraining/training_ops/cpu/aten_ops/aten_op.h"
+#include "orttraining/training_ops/cpu/aten_ops/aten_op_attr_parser.h"
+#include "orttraining/training_ops/cpu/aten_ops/aten_op_executor.h"
+#include "orttraining/training_ops/cpu/aten_ops/aten_op_config.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+ONNX_OPERATOR_KERNEL_EX(
+    ATenOp, kMSDomain, 1, kCpuExecutionProvider,
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::AllTensorAndSequenceTensorTypes()).ExternalOutputs(),
+    ATenOpBase<false>);
+
+ONNX_OPERATOR_KERNEL_EX(
+    ATenOpGrad, kMSDomain, 1, kCpuExecutionProvider,
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::AllTensorAndSequenceTensorTypes()).ExternalOutputs(),
+    ATenOpBase<true>);
+
+template <bool is_backward>
+ATenOpBase<is_backward>::ATenOpBase(const OpKernelInfo& info) : OpKernel(info) {
+  std::string op_name;
+  ORT_THROW_IF_ERROR(info.GetAttr("name", &op_name));
+  ORT_ENFORCE(aten_ops::ATEN_OPERATORS.find(op_name) != aten_ops::ATEN_OPERATORS.end());
```

Comment:
nit... I think it worth writing an util function for check if aten ops is supported. Such check seems pretty common... 
Something lile
inline bool aten_ops::HasAtenOp(op_name) ? 
or 
inline bool aten_ops::TryGetAtenOp(op_name, op_config) ?